### ISSUE TITLE
Buttons don't appear outside of main pane on startup

### ISSUE THREAD

**Issue by john-shupe-amteck** [shabegom/buttons#67]
I have a table of contents that I keep docked to the left side of my workspace so it is easily accessible.

![image](https://user-images.githubusercontent.com/67120753/129909798-db1faf62-52b3-4938-b5f4-168f6899ae04.png)

Upon opening obsidian, the buttons are not initially loaded if the file is not in the main section. 

![image](https://user-images.githubusercontent.com/67120753/129910066-3310c7f9-e770-4ed6-a539-95f65104e7a7.png)

It is easy enough to close the page and reload it but I wanted to at least put this here.

**Comment by john-shupe-amteck** [comment]
I did see that there were other reports of similar bugs. I have updated to the latest release and tried the solutions in those issues but the problem still persists.

**Comment by shabegom** [comment]
Yep, there are problems when buttons are in the side-panel as the plugin isn't expecting it. I've been wanting to add side-panel support for some time, but haven't had the bandwidth.
